### PR TITLE
ESRI GDB Support [Don't Merge]

### PR DESCRIPTION
**Pull request by ingalls**
Refs: https://github.com/openaddresses/openaddresses/issues/1720
Refs: https://github.com/openaddresses/openaddresses/pull/1807

### CODE REVIEW

**Comment by ingalls** [comment]
Currently working with `openadd-process-one` if the file is named `name.gdb.zip`

### TODO: 
- [ ] Add a bunch of tests
- [ ] Test with LA County source

cc/ @migurski

File: openaddr/conform.py [hunk 1]
```diff
@@ -431,14 +445,12 @@ def find_source_path(source_definition, source_paths):
...
             return None
-
-
     else:
         _L.warning("Unknown source type %s", conform["type"])
         return None
 
 class ConvertToCsvTask(object):
-    known_types = ('.shp', '.json', '.csv', '.kml')
+    known_types = ('.shp', '.json', '.csv', '.kml', 'gdb')
```
Comment (iandees) [inline openaddr/conform.py]: Needs a `.` there.

Comment (iandees) [inline openaddr/conform.py]: Well, not "needs", but "should put a dot before gdb to be consistent".

**Comment by ingalls** [comment]
Going to try a slightly different approach in gdb2 branch

**Pull request closed without merging**